### PR TITLE
feat: Publish multi-arch image for `files-artifacts-expander`

### PR DESCRIPTION
**Pull request by gbouv**
## Description:
files-artifacts-expander will now be published for Mac M1 as well as AMD64

## Is this change user facing?
Technically YES, but not sure they really care about this

### CODE REVIEW

**Comment by gbouv** [comment]
Starting with only file expander for now, will add others once we have one successful release with this one

File: scripts/docker-image-builder.sh [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+echo "Docker file located at '${dockerfile_filepath}' in directory '${dockerfile_dirpath}'"
+
+image_tags_concatenated=""
+for image_tag in ${image_tags}; do
+  echo "Image will be tagged with: '${image_tag}'"
+  image_tags_concatenated=" -t ${image_tag}${image_tags_concatenated} "
+done
+
+# Build Docker image
+kurtosis_docker_builder="kurtosis-docker-builder"
```
Comment (h4ck3rk3y) [inline scripts/docker-image-builder.sh:61]: maybe some formatting can be fixed here and have some new lines between the different blocks!

File: scripts/docker-image-builder.sh [hunk 2]
```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env bash
```
Comment (h4ck3rk3y) [inline scripts/docker-image-builder.sh:1]: for this mostly trusting you; i am a "linter" and I just checked that args are quoted!

File: .circleci/config.yml [hunk 3]
```diff
@@ -817,21 +817,19 @@ jobs:
           version: "<< pipeline.parameters.docker-engine-version>>"
       - run: echo "${DOCKER_PASSWORD}" | docker login -u ${DOCKER_USERNAME} --password-stdin
 
-      - run: core/files_artifacts_expander/scripts/build.sh
+      - run: core/files_artifacts_expander/scripts/build.sh true
```
Comment (h4ck3rk3y) [inline .circleci/config.yml:820]: cool we are doing this so that we aren't building the image multiple times

**Review by h4ck3rk3y — APPROVED** [review]

File: .circleci/config.yml (hunk 3 above)
Comment (gbouv) [reply .circleci/config.yml:820]: Exactly 👍

File: scripts/docker-image-builder.sh (hunk 2 above)
Comment (gbouv) [reply scripts/docker-image-builder.sh:1]: ACK, not a bash pro neither, but tested everything I could test I think, will see how it runs on the CI

File: scripts/docker-image-builder.sh (hunk 1 above)
Comment (gbouv) [reply scripts/docker-image-builder.sh:61]: ACK, will add blank lines and some comments

**Pull request merged**